app/routes/shop/index.js

```diff
@@ -7,33 +7,88 @@ const {
   Route
 } = Ember;
 
+const WIDTH = 14;
+const HEIGHT = 18;
+
 const styles = v({
-  shopRoute: {
-    '@composes': [c.liquidInner],
-    top: 0
-  },
-  itemContainer: {
-    '@composes': [c.white],
-    backgroundRepeat: 'no-repeat',
-    backgroundSize: 'cover',
-    backgroundPosition: 'top center',
-    minHeight: '400px',
-    'h3': {
-      '@composes': [c.white],
-      fontWeight: 300,
-      textAlign: 'center',
-      fontSize: '2rem',
-      textDecoration: 'none',
-      ':first-of-type': {
-        marginTop: '20%',
+  shopBlock: {
+    '@composes': [
+      c.col12,
+      c.mdCol4,
+      c.relative
+    ],
+    ':before': {
+      display: 'block',
+      content: `""`,
+      width: '100%',
+      paddingTop: `${(HEIGHT / WIDTH) * 100}%`,
+    },
+    '> .content': {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      cursor: 'pointer',
+      ':hover': {
+        '.shop .overlay': {
+          opacity: 0,
+        },
+        '.shop .image': {
+          opacity: 1,
+        },
+        '.shop .link-wrapper': {
+          color: c.black.color,
+        },
+      },
+    },
+    '.shop': {
```

Comment:
why are u using regular classes here?  can u not do
`class="{{local.shopBlock.shop}}"` in the template?